Serenity-Website/src/ToolTip/ToolTip.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from "react";
+import "./ToolTip.css";
+
+const ToolTip = ({ children, text, duration = 5000 }) => {
```

Comment:
Suggestion: It might be good to not have a duration for the tooltip. As long as the user is hovering over the button, it should continue to show. That way the user won't have to rush to finish reading the text.

Reply:
Okay, true. I was a bit conflicted on making it timed as well. I initially made it visible for as long as the user hovers around it.